.github/workflows/release-t0rn.yml

```
@@ -3,7 +3,7 @@ name: t0rn Rococo Release
 on:
   push:
     tags:
-      - "v[0-9]+\.[0-9]+\.[0-9]+-rc\.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
```

```suggestion
      # This is a GLOB not a regex. Don't add escape sequences to me
      - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
```

I fixed this in my pr, but please can we add a note here so that this doesnt happen again